packages/databyss-ui/modules/Landing/Landing.js

```diff
@@ -1,48 +1,128 @@
 import React from 'react'
 import injectSheet from 'react-jss'
 import classnames from 'classnames'
+import SwitchControl from '../../components/Control/SwitchControl'
 import Content from '../../components/Viewport/Content'
+import LandingBody from '../../components/Viewport/LandingBody'
 import Raw from '../../components/Viewport/Raw'
 import PageHeading from '../../components/Heading/PageHeading'
 import PageSubHeading from '../../components/Heading/PageSubHeading'
 import PageNav from '../../components/Navigation/PageNav'
 import CommaSeparatedList from '../../components/List/CommaSeparatedList'
+import LandingHeading from '../../components/Heading/LandingHeading'
 import ContentHeading from '../../components/Heading/ContentHeading'
 import styles from './styles'
 
-const Landing = ({
-  className,
-  classes,
-  cfList,
-  renderCfItem,
-  contentTitle,
-  children,
-  title,
-  subtitle,
-}) => (
-  <Content className={classnames(className, classes.landing)}>
-    <PageHeading>
-      <Raw html={title} />
-    </PageHeading>
-    {subtitle && (
-      <PageSubHeading>
-        <Raw html={subtitle} />
-      </PageSubHeading>
-    )}
-    {cfList && (
-      <PageNav ariaLabel="compare with">
-        <CommaSeparatedList opener="[cf." closer="]">
-          {cfList.map(cf =>
-            React.cloneElement(renderCfItem(cf), { key: cf.id })
+class Landing extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      borderBottom: false,
+      contentLoaded: false,
+      headerLoaded: false,
+    }
+  }
```

Comment:
here is where you instantiate the header ref, like `this.headerRef = React.createRef()`